src/components/buttons/icon-button/icon-button.js

```diff
@@ -42,10 +40,6 @@ export const IconButton = props => {
   const isActive = props.isToggleButton && props.isToggled;
   return (
     <div
```

Comment:
Is it also possible to get rid of this `div` altogether and apply the styles to the AccessibleButton component?

Reply:
giving it a shot now

Reply:
okay removed it, and VRTs passed 🤷‍♂

Reply:
Nice!
I always found it weird that we had an AccessibleButton, which already deals with the mouse events, but then wrap it on a dull `div` which then reimplemented those mouse events with the HoC... I guess there must have been some other reason before, but something that got refactored? 🤔

 It makes sense that it can be removed given how useless it seemed at the moment 🤷‍♂

Reply:
do you want to approve then 🧐